src/createServer.js

```diff
@@ -4,0 +1,58 @@
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const requestUrl = new URL(req.url, `http://${req.headers.host}`);
+    const text = requestUrl.pathname.substring(1);
+    const toCase = requestUrl.searchParams.get('toCase');
+
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+      });
+    } else if (!CASES.includes(toCase)) {
+      errors.push({
+        message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
+      });
+    }
+
+    if (errors.length) {
+      res.statusCode = 400;
+      res.end(JSON.stringify({ errors }));
+
+      return;
+    }
+
+    const result = convertToCase(text, toCase);
```

Comment:
Ensure that the `convertToCase` function is correctly handling the input `text` and `toCase`. If `convertToCase` is not defined properly in the imported module, it could lead to errors or unexpected results.